token-metadata/js/test/update.test.ts

```diff
@@ -1327,7 +1329,7 @@ test('Update: Holder Authority Type Not Supported', async (t) => {
   await updateTx.assertError(t);
 });
 
-test('Update: Update pNFT Config with locked token', async (t) => {
+test('Update: Cannot Update pNFT Config with locked token', async (t) => {
```

Comment:
Maybe this is just too much of a nitpick detail, but this test delegates and locks and then it fails because it won't update pNFT config due to delgate.

Should this or another test exist that only delegates and then fails because it is delegated?  Since that is kind of the new requirement specifically.

Reply:
Yeah, probably. I just modified this to fix it given the updates in v1.8.0.